xcube_cds/store.py

```diff
@@ -759,17 +758,23 @@ def get_type_specifiers_for_data(self, data_id: str) -> Tuple[str, ...]:
         return TYPE_SPECIFIER_CUBE,
 
     def get_data_ids(self, type_specifier: Optional[str] = None,
-                     include_titles: bool = True) \
-            -> Iterator[Tuple[str, Optional[str]]]:
-        if not self._is_type_specifier_satisfied(type_specifier):
-            # If the type specifier isn't compatible, return an empty iterator.
-            return iter(())
-        return iter(
-            (data_id,
-             self._handler_registry[data_id].
-             get_human_readable_data_id(data_id) if include_titles else None)
-            for data_id in self._handler_registry
-        )
+                     include_attrs: Container[str] = None) -> \
+            Union[Iterator[str], Iterator[Tuple[str, Dict[str, Any]]]]:
+
+        if self._is_type_specifier_satisfied(type_specifier):
+            # Only if the type specifier isn't compatible
+            return_tuples = include_attrs is not None
+            # TODO: respect names other than "title" in include_attrs
```

Comment:
Why is this necessary?

Reply:
Because other attributes may be requested by some client.